sqlmesh/migrations/_0000.py

```diff
@@ -0,0 +1,8 @@
+"""All migrations should be named _XXXX.py, they will be executed sequentially.
```

Comment:
Can we mimic flyway here and following the following format:
```
V<index>__<description>.py
```
Not sure why follow this weird 4 digit format.

Reply:
so that it's lexically sortable, it's used in https://sequel.jeremyevans.net/

i could do the description but i've never found those helpful